src/components/AiMarketplace/MainSection/ServiceCollection/CardGroup/index.js

```diff
@@ -4,52 +4,57 @@ import { withStyles } from "@material-ui/styles";
 import ListView from "./ListView";
 import StyledCard from "./StyledCard";
 
+import CardImg from "../../../../../assets/images/dummy-card.png";
+
 import { useStyles } from "./styles";
```

Comment:
Will club all the userImports together. So need no new lines from line4 to line9.